binding.gyp

```diff
@@ -17,7 +17,6 @@
         }, {
           'conditions': [
             ['OS=="win"', {
-              'msbuild_toolset': 'v140',
               'defines': ['ZMQ_STATIC'],
```

Comment:
Why is it OK to remove this?

Reply:
This allows us to build with any Visual Studio and not just VS 2015. Node-gyp itself is an abstraction that handles finding the compiler. There is no need to directly specify the toolchain version and de-abstract the build system!

The error with keeping this:
```js
C:\Program Files (x86)\Microsoft Visual Studio\2019\Preview\MSBuild\Microsoft\VC\v160\Microsoft.CppBuild.targets(439,5)
: error MSB8020: The build tools for Visual Studio 2015 (Platform Toolset = 'v140') cannot be found. To build using the
 v140 build tools, please install Visual Studio 2015 build tools.  Alternatively, you may upgrade to the current Visual
 Studio tools by selecting the Project menu or right-click the solution, and then selecting "Retarget solution". [C:\Us
ers\aminy\Documents\GitHub\JavaScript\zeromq.js\build\zmq.vcxproj]
```